docs/reference/elasticsearch/rest-apis/retrievers/linear-retriever.md

```diff
@@ -37,10 +37,7 @@ Combining `query` and `retrievers` is not supported.
     
     When used with the [multi-field query format](../retrievers.md#multi-field-query-format) (`query` parameter), normalizes scores per [field grouping](../retrievers.md#multi-field-field-grouping).
     Otherwise serves as the default normalizer for any sub-retriever that doesn't specify its own normalizer. Per-retriever normalizers always take precedence over the top-level normalizer.
```

Comment:
```suggestion

    Otherwise this serves as the default normalizer for any sub-retriever that doesn't specify its own normalizer. Per-retriever normalizers always take precedence over the top-level normalizer. {applies_to}`stack: ga 9.2`
```

Reply:
Let's break this onto its own line for visibility, I found it a little messy when exploring the URL preview: https://github.com/elastic/elasticsearch/pull/134216#issuecomment-3258211557